draft-ietf-quic-transport.md

```diff
@@ -2353,7 +2351,8 @@ processed successfully.  The timer is also restarted when sending an
 ack-eliciting packet (see {{QUIC-RECOVERY}}), but only if no other ack-eliciting
 packets have been sent since last receiving a packet.  Restarting when sending
 packets ensures that connections do not prematurely time out when initiating new
-activity.
+activity.  An endpoint should be careful to avoid an idle timeout if it is idle
```

Comment:
You use "should", which is a little sketchy.

> An endpoint might need to send packets to avoid an idle timeout if it is unable to send application data due to being blocked on flow control limits (see {{data-flow-control}}).

Reply:
You can thank @janaiyengar for the sentence. :) Either way, I do prefer Jana's text over "might need to send", though both seem like they are just trying to skate around using normative language.

Reply:
Not so much.  We can't require anything here as there isn't any interoperability hazard we're avoiding.  The goal is to make the consequence of certain choices clear.